master/buildbot/schedulers/forcesched.py

```diff
@@ -616,6 +617,10 @@ def __init__(self, name, builderNames,
             config.error("ForceScheduler name must not be empty: %r " %
                          name)
 
+        if not identifiers.ident_re.match(name):
+            config.error("ForceScheduler name must be an identifier: %r " %
```

Comment:
Is trailing whitespace in the string is necessary?

Can you fix string above too, please?